users/serializers.py

```diff
@@ -32,3 +32,9 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
 
         return user
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'email']
```

Comment:
Okay, it is right now. Thank you for acknowledging my comment and improving the code.